android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -348,8 +348,9 @@ public void endAllCalls() {
             return;
         }
 
-        Map<String, VoiceConnection> currentConnections = VoiceConnectionService.currentConnections;
-        for (Map.Entry<String, VoiceConnection> connectionEntry : currentConnections.entrySet()) {
+        ArrayList<Map.Entry<String, VoiceConnection>> connections =
+            new ArrayList<Map.Entry<String, VoiceConnection>>(VoiceConnectionService.currentConnections.entrySet());
+        for (Map.Entry<String, VoiceConnection> connectionEntry : connections) {
             Connection connectionToEnd = connectionEntry.getValue();
```

Comment:
Ne pourrais-tu pas faire aussi?
`Connection connectionToEnd = connectionEntry.clone().getValue();`

Reply:
I've got this error:
```
RNCallKeepModule.java:379: error: cannot find symbol
            Connection connectionToEnd = connectionEntry.clone().getValue();
                                                        ^
  symbol:   method clone()
  location: variable connectionEntry of type java.util.Map.Entry<java.lang.String,io.wazo.callkeep.VoiceConnection>
```